src/protocol/thread/value.ts

```diff
@@ -127,6 +127,14 @@ export class ValueFront {
     return [];
   }
 
+  previewPromiseState() {
+    if (this.hasPreview()) {
+      // Older recordings did not set promiseState so this could be null.
+      return this._object!.preview!.promiseState || null;
+    }
+    return null;
```

Comment:
Could be simplified to `return this._object?.preview?.promiseState || null;`

Reply:
Yeah I can change it. It's this way because I copy-pasted `previewContainerEntries` and never bothered to reformat it.